py/dml/messages.py

```diff
@@ -820,6 +820,13 @@ def log(self):
         if self.othersite:
             self.print_site_message(self.othersite, "conflicting declaration")
 
+class EAUNKDIM(DMLError):
```

Comment:
Class name is incorrect, its not the DIM thats unknown its the SIZE

Reply:
`EAUNKDIMSIZE`, then?

Reply:
Seems good